CONTRIBUTING.md

```diff
@@ -54,20 +60,40 @@ Refactoring:
 refactor/branch-name
 ```
 
-## Pull Request process
+## Pull request process
 
 Use the PR title to summerise changes; and the body to give further detail on specific changes (reference associated issues here).
 
-All PRs must be reviewed and approved by at least one other team member before being merged. A review means running the test suite, running the code, and ensuring the code works as expected.
+All PRs must be reviewed and approved by at least two other team members before being merged. A review means running the test suite, running the code, and ensuring the code works as expected.
 
-When completely a PR make sure you use 'Squash & Merge'.
+If you think certain parts of your changes need particular attention, please leave a comment on the PR highlighting these particular changes.
+
+Please do not make your PR too large - if splitting it up into two PRs makes it easier for the reviewer, then please create new branches / PRs and cherry-pick the relevant commits.
+
+When completely a PR make sure you use **'Squash & Merge'.**
 
 ## Creating issues
 
 All issues need to be approved by one other contributer before work on that issue can begin.
-You can create a new issue to document bugs (including documentation issues) or request new features that were not included in the initial list of
-issues.  
+
+You can create a new issue to document bugs (including documentation issues) or request new features that were not included in the initial list of issues. 
+
 For a new feature request, the issue should describe the new feature and why it is needed.  
 For bug reports, provide as much information as you can so that the person who fixes the bug will be able to reproduce it.  
+
 For all new issues, please make sure you check the open issues first to avoid creating duplicate issues.  
 If more than one person works on an issue, ensure this is documented in the issue and pull request comments.
+
+## Tests
+All  additions or modifications to code should include associated tests (an automated test suite should be created).
+Please see [Puppeteer](https://github.com/Mafia-Hands/Mafia/wiki/Frontend-Tech) and [backend testing](https://github.com/Mafia-Hands/Mafia/wiki/Backend-Testing-Infrastructure) for more details on how to create tests within our codebase.
```

Comment:
The puppeteer link is not functioning. Maybe just link to the Frontend page or the puppeteer markdown file?

Reply:
Fixed, thanks :)